src/content/overview/introduction.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Introduction to Giant Swarm
+description: Giant Swarm give platform engineering teams the ability to build cloud-native developer platforms and operate them without much hassle.
+weight: 10
+menu:
+  main:
+    parent: overview
+user_questions:
+  - What is a cloud-native developer platform?
+  - How does Giant Swarm help engineering teams?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-horizon
+last_review_date: 2024-03-12
+---
+
+Giant Swarm's mission is to empower our customers to build their own cloud-native developer platforms. Most engineering teams today want to build a [platform](https://tag-app-delivery.cncf.io/whitepapers/platforms/#what-is-a-platform) and there are [many reasons](https://tag-app-delivery.cncf.io/whitepapers/platforms/#why-platforms) why a unified platform actually makes sense, but the teams should also be aware that the complexity in today's cloud-native landscape is overwhelming.
```

Comment:
Giant Swarm's mission is to empower our customers to build their own cloud-native [developer platform](https://tag-app-delivery.cncf.io/whitepapers/platforms/#what-is-a-platform). As engineering teams grow and their environments expand, they often face exponentially growing complexity within the cloud-native ecosystem. This can be overwhelming. We believe that in most cases, having a [unified platform](https://tag-app-delivery.cncf.io/whitepapers/platforms/#why-platforms) plays a crucial role in keeping or gaining control over the entropic nature of such complex environments, while also significantly enhancing the users experience in using them.

Got a bit carried away and ended up rewriting the whole paragraph. Maybe you want to use something :)